data-raw/prepare_data_for_model.R

```diff
@@ -69,6 +109,23 @@ stream_temperature <- monthly_mean_temperature %>%
 
 dimnames(stream_temperature) <- list(cvpia_watershed, month.abb, 1980:2000)
 
+return(stream_temperature)
+}
+
+stream_temp_2008_2009 <- generate_stream_temperature(monthly_mean_temperature_2008_2009)
+stream_temp_2018_2019 <- generate_stream_temperature(monthly_mean_temperature_2018_2019)
+
+# Check how new modeled results compare to old calsim results
+# TODO more digging into temp modeling on these ones
```

Comment:
Quick summary about the differences in modeled temps vs of temps for San Joaquin, Stanislaus, Tuolumne, and Merced. I am a bit concerned by Tuolumne and Stanislaus where there are some big degree differences

Reply:
bring these up with Mark